### ISSUE TITLE
How to add InfoPlist field?

### ISSUE THREAD

**Issue by kingsword09** [mobanisto/pinpit-gradle-plugin#46]
I want to add CFBundleURLSchemes of CFBundleURLTypes field to add custom scheme for DeepLink, how can I add this field for packaging.

**Comment by kingsword09** [comment]
@sebkur

**Comment by sebkur** [comment]
Hey @kingsword09, thanks for your request. Could you provide a little more context please? What's your goal? I have google the keywords you provided and it looks like this concerns macOS, right? What do these schemes do when added to the configuration usually?

**Comment by kingsword09** [comment]
> Hey @kingsword09, thanks for your request. Could you provide a little more context please? What's your goal? I have google the keywords you provided and it looks like this concerns macOS, right? What do these schemes do when added to the configuration usually?

Yes, this is for Mac OS, and this is what it's for [custom url scheme](https://developer.apple.com/documentation/xcode/defining-a-custom-url-scheme-for-your-app).

**Comment by kingsword09** [comment]
I currently achieve the effect I want by replacing the Info.plist file.
